<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QDialog" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>640</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>240</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload Image</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="imagePath">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>210</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>540</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Access List</string>
   </property>
  </widget>
  <widget class="QListView" name="ServerImageList">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>256</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="MyImageList">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>256</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Accessible Images on the server:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Your Images:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ViewsNo">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>480</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>List of users on the server:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>450</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Views</string>
   </property>
  </widget>
  <widget class="QListView" name="UsersList">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="checkuser">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>350</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Refresh">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newUser">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>480</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>450</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>350</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Check User</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ImageID">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>480</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Image ID</string>
   </property>
  </widget>
  <widget class="QToolButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>210</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ViewOwnImage">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>380</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcome">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumBarunGothic</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Welcome</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcome_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>NanumBarunGothic</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Hello</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Home</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Home</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
